sources/Launcher/DYNMarginCalculationLauncher.cpp

```diff
@@ -728,26 +716,31 @@ MarginCalculationLauncher::synchronizeSuccesses(const std::vector<bool>& success
 
 void
 MarginCalculationLauncher::findOrLaunchLoadIncrease(const boost::shared_ptr<LoadIncrease>& loadIncrease,
-    const double variation, const double minVariation, const double maxVariation, const double tolerance, SimulationResult& result) {
+                                                    const double variation,
+                                                    const double minVariation,
+                                                    const double maxVariation,
+                                                    const double tolerance,
+                                                    LoadIncreaseResult& loadIncreaseResult) {
   TraceInfo(logTag_) << DYNAlgorithmsLog(VariationValue, variation) << Trace::endline;
 
   auto found = loadIncreaseStatus_.find(variation);
   if (found != loadIncreaseStatus_.end()) {
-    result = importResult(computeLoadIncreaseScenarioId(variation));
+    const SimulationResult importedResult1 = importResult(computeLoadIncreaseScenarioId(variation));
+    loadIncreaseResult.setResult(importedResult1);
     TraceInfo(logTag_) << DYNAlgorithmsLog(LoadIncreaseResultsFound, variation) << Trace::endline;
     return;
   }
 
   if (mpi::context().nbProcs() == 1) {
     inputs_.readInputs(workingDirectory_, loadIncrease->getJobsFile());
-    launchLoadIncrease(loadIncrease, variation, result);
+    const SimulationResult result = launchLoadIncrease(loadIncrease, variation);
+    loadIncreaseResult.setResult(result);
 
     // Hack to add 0. if the load increase is below 50. as we know we never did 0. in the first place
     if (variation < 50. && !result.getSuccess() && loadIncreaseStatus_.find(0.) == loadIncreaseStatus_.end()) {
       TraceInfo(logTag_) << DYNAlgorithmsLog(VariationValue, 0.) << Trace::endline;
-      SimulationResult result0;
       inputs_.readInputs(workingDirectory_, loadIncrease->getJobsFile());
-      launchLoadIncrease(loadIncrease, 0., result0);
+      const SimulationResult result0 = launchLoadIncrease(loadIncrease, 0.);
```

Comment:
unused?

Reply:
result0 is used at the line below + I reverted my changes anyway